### ISSUE TITLE
Tables: Currently selected pagination not indicated to screen readers

### ISSUE THREAD

**Issue by szet0018** [wet-boew/wet-boew#9664]
Screen reader users cannot tell which pagination link is currently selected
e.g.
```html
<a class="paginate_button current" aria-controls="wb-auto-1" data-dt-idx="1" tabindex="0" href="#wb-auto-1" aria-pressed="true"><span class="wb-inv">Jump to: Page </span>1</a>
```

Potential Solution
Add aria-current="page" to the currently selected pagination link
OR
Add visually hidden text "current" to indicate it is the currently selected link e.g. <span class="wb-inv">Current Page </span>1

**Comment by EricDunsworth** [comment]
@szet0018 FYI I've edited the OP to wrap your HTML code sample into a real code block. I presume that's what you intended to show off (as opposed to a random "rendered" link with most of its markup filtered-out).

**Comment by EricDunsworth** [comment]
[A similar issue was reported in DataTables' forum earlier this year](https://datatables.net/forums/discussion/75174). It led to ``aria-current="page"`` getting added starting in [version 13.2.0](https://cdn.datatables.net/1.13.2/). DataTables is the backbone of WET's tables plugin.

#9700 should bring WET's DataTables dependency up to-date :).

**PS:**
I personally feel like ``aria-current="true"`` might've been more appropriate for pagination (to keep ``page`` reserved as an identifier for the main document). But I guess ``aria-current="page"`` is better than nothing.

Btw WET's tables plugin was initially designed to use ``aria-role="button"`` and ``aria-pressed="true"`` to denote the current pagination page (no idea why - maybe ``aria-current`` didn't exist back then?). But it looks ``aria-role="button"`` was accidentally-removed at some point and broke it :(.
